langchain/llms/vertexai.py

```diff
@@ -50,10 +50,21 @@ def _default_params(self) -> Dict[str, Any]:
         }
         return {**base_params}
 
+    @property
```

Comment:
Could you update the docstrings on top_k and top_p saying they are ignored for code generation models?